frontend/public/components/events.jsx

```diff
@@ -269,7 +269,7 @@ export const EventsList = (props) => {
             selected={Array.from(selected)}
             clearSelection={clearSelection}
           />
-          <Dropdown
+          <ConsoleSelect
```

Comment:
Event Type not updated with selected value

https://github.com/user-attachments/assets/36f89813-18b4-4a62-834d-92be273f40c2

Reply:
all of the dropdowns not updating with a selected value was a result of some oversimplification I made, which has since been reverted. these should work now